src/codegen/cfg.rs

```diff
@@ -2122,6 +2106,39 @@ impl Contract {
 }
 
 impl Namespace {
+    /// Determine whether a function should be included in the dispatacher and metadata,
+    /// taking inheritance into account.
+    ///
+    /// `function_no` is optional because default constructors require creating a CFG
+    /// without any corresponding function definition for default constructors.
```

Comment:
```suggestion
    /// `function_no` is optional because default constructors require creating a CFG
    /// without any corresponding function definition.
```